### ISSUE TITLE
Docrine transport might not work properly with Postresql

### ISSUE THREAD

**Issue by db306** [Guikingone/SchedulerBundle#229]
The following configuration works for MySQL (8.0) but not when using Postgresql (11.5) 

```yaml
scheduler_bundle:
    transport:
        dsn: 'doctrine://default?execution_mode=first_in_first_out'
```

When using Postgresql this output the following error when rebooting:

```
SQLSTATE[0A000]: Feature not supported: 7 ERROR:  FOR SHARE is not allowed with aggregate functions
```

We can close this ticket if others have managed to make this work on Postgres

Thanks

**Comment by Guikingone** [comment]
Hi @db306 👋🏻 

Pretty strange, I'm not sure about the sentence: 

> When using Postgresql this output the following error when rebooting

Could your more specific about the `rebooting` phase? 🤔

**Comment by chapterjason** [comment]
Just wanted to use it in a project and directly hit this issue. Is there anything I can help with?

postgresql log:
```
2022-04-09 20:04:53.038 UTC [951] ERROR:  FOR SHARE is not allowed with aggregate functions
2022-04-09 20:04:53.038 UTC [951] STATEMENT:  SELECT COUNT(DISTINCT t.id) FROM _symfony_scheduler_tasks t FOR SHARE
```

**Comment by Guikingone** [comment]
Hi @chapterjason 👋🏻 

Yes, it seems that Postgres does not support the lock of a row, the usage here was to prevent any parallel execution of the task when called via the worker but the worker already lock the task using `Lock`.

The only solution here is to remove the current call of `$this->driverConnection->getDatabasePlatform()->getReadLockSQL()`, I'm gonna fix it.

**Comment by Guikingone** [comment]
The PR https://github.com/Guikingone/SchedulerBundle/pull/233 tries to fix the issue for the whole bridge, could you test it when you have a moment? 

Thank you and sorry for the delay 🙁

**Comment by chapterjason** [comment]
@Guikingone Take your time, you are doing great work here! :)

The usage has unfortunately dissolved since an external API is not as usable as I thought. But I like to test it in a demo project.

**Comment by Guikingone** [comment]
Hi @chapterjason and @db306 👋🏻 

The fix will be released via `0.9` (soon), it's already available via `dev-main` if you need it faster (the branch is considered stable).

Thanks again for the issue and sorry for the delay 🙂
